extensions/api/api-core/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This module should be declared as dependency of `control-api` and `observability-api` modules

Reply:
The upcoming feature of configurable jetty contexts (#601) will allow that different APIs are exposed under different ports/context paths, so the `ExceptionMapper` will have to be registered explicitly in every one of these. Declaring the dependency onto `api-core` will only register it for the default context.

Slightly unrelated: `control-api` and `datamanagement-api` (#542) have a considerable overlap, so I think it makes sense to collate them into one API and expose them on the same context.


Also, for the `observability-api` I think this mapper really doesn't provide much value.